archive/2025-08-13-sloppy-ai-defenses-take-cybersecurity-back-to-the-1990s-researchers-say-sc-media.md

```diff
@@ -0,0 +1,319 @@
+
+The Black Hat briefings kicked off Aug. 6 with a presentation by Rebecca Lynch and Rich Harang of Nvidia, who detailed how easy it is to feed malicious information to an LLM or an AI agent and what you can do to mitigate (but never quite eliminate) the risk.
+
+Lynch, an offensive security researcher, explained that to alter the output, you have to poison the input. Because many, if not all, LLMs have trouble telling the difference between prompts and data, it's easy to perform the AI equivalent of [SQL injection](https://www.scworld.com/resource/how-to-fix-recent-sql-injection-vulnerabilities-and-avoid-future-attacks) upon them.
+
+"The real question is where untrusted data can be introduced," she said. But fortunately for attackers, she added many AIs can retrieve data from "anywhere on the internet."
+
+Designers of LLM and agentic AI models place filters at various points in the information flow to choke off malicious inputs, but Lynch showed that they sometimes miss a few spots. She ran through real-world examples of [attacks on Microsoft Copilot](https://www.scworld.com/news/microsoft-365-copilot-zero-click-vulnerability-enabled-data-exfiltration) and the open-source data-analysis LLM PandasAI, then demonstrated proof-of-concept [attacks on the AI-powered development tool Cursor.](https://www.scworld.com/news/cursor-flaw-risks-rce-from-prompt-injections-on-mcp-server-researchers-say)
+
+Lynch said she disclosed the issues to the makers of Cursor, but they didn't mitigate the problems. opting instead to add some optional safeguards.
+
+"So what do we do about securing these agents?" asked Harang.
+
+Users and developers of AIs need to use traditional means like sanitizing inputs and restricting access, he said, and he added three new rules.
+
+First, assume prompt injection. [As in zero trust](https://www.scworld.com/perspective/four-ways-to-build-a-zero-trust-program-for-the-ai-world), you should assume your AI can be hacked.
+
+"Design your system to assume the LLM is vulnerable and that it will hallucinate and do dumb things," Harang said.
+
+Second, "if the LLM can see it, the attacker can use it."
+
+Third, remember this maxim: "Once tainted, always untrusted. Once the attacker gets a hook in, everything is compromised because the model will ingest and reuse the malicious data."
+
+Or, as Lynch put it: "As long as an adversary can get their data into your agent, you have to assume that nothing downstream can be trusted."
+
+## All you need to do is ask nicely
+
+Zenity's co-founder and CTO Michael Bargury and his colleague, threat researcher Tamir Ishay Sharbat, explained how the Zenity team pwned Microsoft Copilot Studio, OpenAI ChatGPT, Google Gemini, Salesforce Einstein and Cursor, sometimes using one model to attack another.
+
+We couldn't type quickly enough to get all the details in their presentation, but blog posts about several of the attacks methods are on [the Zenity Labs website](https://labs.zenity.io/p/hsc25).
```

Comment:
Paul is right. We fitted 90 minutes of content into 40 a minute talk with just the gists. 90 minutes director's cut coming up!